### ISSUE TITLE
web3 is not connecting to Rinkeby provider in Firefox

### ISSUE THREAD

**Issue by troggy** [leapdao/bridge-ui#47]
Looks like CORS issue.

Probably related: https://github.com/ethereum/web3.js/issues/1803

Firefox 63.0b13 

### Delivery
- make Bridge UI work in latest Firefox/Brave. Probably, it is as easy as downgrading web3 to beta.33, although I expect some headache with @types/web3 in this case.

### Bounty setup

Value: 165 DAI

gardener: @troggy / 0%
worker: @troggy / 90%
reviewer: @johannbarbie  / 10%

**Comment by sunify** [comment]
I've upgraded web3 to latest beta. beta.33 contains wrong typings and that leads to a lot of typescript errors.

I'll fix web3 and send PR

**Comment by troggy** [comment]
👍
anyway, it didn't work for me in Firefox lately

**Comment by troggy** [comment]
Doesn't work in Brave as well.

Same error

```
Access to fetch at 'https://rinkeby.infura.io/' from origin 'http://localhost:1234' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: Redirect is not allowed for a preflight request.
```

**Comment by troggy** [comment]
Brave has different reasons :(

https://github.com/brave/brave-browser/issues/2252

**Comment by troggy** [comment]
closed via #92
